### DISCUSSION TITLE
Array of objects and single objects on prop field not working

### DISCUSSION THREAD

**Discussion by victorlmneves** [histoire-dev/histoire#185] [Q&A] [answered]
When passing array of objects or single objects on the prop field it shows a warning icon on the side and when putting the cursor over it shows a tooltip saying "JSON error"
![image](https://user-images.githubusercontent.com/17341789/179715959-f4e2d743-5ac0-4d44-b1a1-588e301562db.png)

**Comment by Akryum** [comment, marked as answer]
You need to type valid JSON

Le mar. 19 juil. 2022, 11:23, Victor Neves ***@***.***> a
écrit :

**Reply by victorlmneves** [reply]
Upsss, Now I see that has a comma at the end :/
And also that I'm using single quotes instead of double quotes
